mne/time_frequency/tests/test_tfr.py

```diff
@@ -430,6 +430,47 @@ def test_plot():
     plt.close('all')
 
 
+def test_plot_joint():
+    """Test TFR plotting."""
+    import matplotlib.pyplot as plt
+
+    raw = read_raw_fif(raw_fname)
+    times = np.linspace(-0.1, 0.1, 200)
+    n_freqs = 3
+    nave = 1
+    rng = np.random.RandomState(42)
+    data = rng.randn(len(raw.ch_names), n_freqs, len(times))
+    tfr = AverageTFR(raw.info, data, times, np.arange(n_freqs), nave)
+
+    tfr.plot_joint(picks=[0, 1], title='title', colorbar=True,
+                   aggregate='mean')
+    plt.close('all')
+
+    tfr.plot_joint(picks=[0, 1], title='auto', colorbar=True,
+                   aggregate='rms')
+    plt.close('all')
+
+    tfr.plot_joint(picks=[0, 1], title='auto', colorbar=True,
+                   aggregate=None)
+    plt.close('all')
+
+
+    fig = tfr.plot_joint(exclude=[raw.ch_names[0]])
+    fig.canvas.key_press_event('up')
+    fig.canvas.key_press_event(' ')
+    fig.canvas.key_press_event('down')
+    fig.canvas.scroll_event(0.5, 0.5, -0.5)  # scroll down
+    fig.canvas.scroll_event(0.5, 0.5, 0.5)  # scroll up
```

Comment:
Are these events supposed to do something? If not, we can probably live without them.